nuget/roundhouse/roundhouse.nuspec

```diff
@@ -12,6 +12,15 @@
     <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <tags>roundhouse db migration database migrator chucknorris</tags>
-	<iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
+    <dependencies />
+	  <iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
   </metadata>
+  <files>
+    <file src="$mergedExe$" target="bin\" />

```

Comment:
What do you think of moving this to the tools directory or a subdirectory of that?

Reply:
I don't know what chocolatey expects. The nuget of the exe is basically for Choco, isn't it?

Reply:
It doesn't matter where it is, but what the consistent approach has been for many packages is a subfolder of tools.

Reply:
So the `rh.exe` itself should be in a subfolder of `tools`? and then it's the job of `chocolateyInstall.ps1` to do what it wants with it (copy to Program Files or AppData or whatever)?

Reply:
I'm not sure that chocolateyInstall.ps1 is even necessary.

Reply:
The `install.ps1` needs to go away.

Reply:
With just binaries in the package (no scripts), you can run these commands in succession:

* `choco install roundhouse`
* `rh`

Chocolatey automatically shims executables found in packages and puts them into a folder that is on the PATH (`$env:ChocolateyInstall\bin`).

Reply:
Nice. I now know more why i love choco ;)
So, skip the chocolateyInstall.ps1. Where do we put rh.exe? Tools? root?

Reply:
My vote for consistency is the tools folder.

Reply:
So we end up with just this in the Nugget (I love calling them nuggets), then?

![image](https://user-images.githubusercontent.com/1628994/31964192-8c2c6cbc-b903-11e7-8f8d-c6708fd93deb.png)